core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.formatinstant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.UnsupportedTemporalTypeException;
+
+import org.joda.time.format.DateTimeFormat;
+import org.junit.Test;
+
+public class FormatInstantUnitTest {
+    
+    private static final String PATTERN_FORMAT = "dd.MM.yyyy";
+
+    @Test
+    public void givenInstant_WhenUsingDateTimeFormatter_thenFormat() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(PATTERN_FORMAT)
+            .withZone(ZoneId.systemDefault());
+
+        Instant instant = Instant.parse("2022-02-15T18:35:24.00Z");
+        String formattedInstant = formatter.format(instant);
+
+        assertThat(formattedInstant).isEqualTo("15.02.2022");
+        
```

Comment:
For this and other methods, remove the blank line before the closing brace

Reply:
done